app/containers/TunesContainer/index.js

```diff
@@ -144,15 +155,15 @@ TunesContainer.defaultProps = {
 };
 
 const mapStateToProps = createStructuredSelector({
-  tunesContainer: makeSelectTunesContainer(),
+  tunesContainer: selectTunesContainer(),
```

Comment:
Are we using this prop ?

Reply:
yes. in a test

Reply:
Since we have individual selectors, let us remove this selector and it's tests.